healthcheck.py

```
@@ -0,0 +1,50 @@
+            self.wfile.write(json.dumps(response).encode())
+        else:
+            self.send_response(404)
+            self.end_headers()
+
+    def log_message(self, format, *args):
+        logger.info(format, *args)
+
+
+def start_server():
+    """Start the healthcheck HTTP server."""
+    server = HTTPServer(("0.0.0.0", PORT), HealthHandler)
```

Ruff (0.15.2)

[error] Possible binding to all interfaces

(S104)